### PR TITLE
Orders show new

### PR DESCRIPTION
**Pull request by reid-andrew**
Code for User Stories 27-30
Added in model tests for Items Model methods
Includes migration to store order/order_item status in database for reference & updating

### CODE REVIEW

File: app/controllers/profile_controller.rb [hunk 1]
```diff
@@ -17,21 +17,18 @@ def edit
   def update
     @user = current_user
     @user.update(user_params)
-    if User.where(email:@user.email).where.not(id:@user.id) != []
-      flash[:failure] = "That email already exists"
-      render :edit
-    elsif @user.save
-      redirect_to '/profile'
-      flash[:error] = "Your profile is updated"
+    if @user.save
```
Comment (aperezsantos) [inline app/controllers/profile_controller.rb:20]: Nice refactor, cleaner.

File: spec/features/profile/orders/show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      updated_order = Order.find(@order_1.id)
+      updated_line_1 = ItemOrder.find(@line_item_1.id)
+      updated_line_2 = ItemOrder.find(@line_item_2.id)
+      updated_line_3 = ItemOrder.find(@line_item_3.id)
+
+      expect(updated_order.status).to eq("Canceled")
+      expect(updated_line_1.status).to eq("Unfulfilled")
+      expect(updated_line_2.status).to eq("Unfulfilled")
+      expect(updated_line_3.status).to eq("Unfulfilled")
+# - Each row in the "order items" table is given a status of "unfulfilled"
```
Comment (aperezsantos) [inline spec/features/profile/orders/show_spec.rb:159]: Let's remember these comments for later

**Review by aperezsantos — APPROVED** [review]
Tests look good. Some comments will need to be taken out.

File: db/migrate/20200412174115_add_status_to_order.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddStatusToOrder < ActiveRecord::Migration[5.1]
+  def change
+    add_column :orders, :status, :string, :default => "Pending"
```
Comment (javier-aguilar) [inline db/migrate/20200412174115_add_status_to_order.rb:3]: Should the default be Pending? 

I know these are the four that an order can have
'pending' means a user has placed items in a cart and "checked out" to create an order, merchants may or may not have fulfilled any items yet
'packaged' means all merchants have fulfilled their items for the order, and has been packaged and ready to ship
'shipped' means an admin has 'shipped' a package and can no longer be cancelled by a user
'cancelled' - only 'pending' and 'packaged' orders can be cancelled

**Review by aperezsantos — APPROVED** [review]

**Review by javier-aguilar — APPROVED** [review]
Everything looks solid

**Pull request merged**